# Goal: corresponds to the "output value" and "context"
float64 y_new
float64[] c_new
---
# Result: corresponds to the new parameters
float64[] x_new
---
# Feedback: as of now we do not use this
bool feedback